dev-packages/ext-scripts/theia-ext.js

```diff
@@ -48,19 +58,26 @@ function run(script) {
 }
 
 (async () => {
+    /** @type {Error | number} */
     let exitCode = 0;
     let extScript = undefined;
     try {
         extScript = getExtScript();
+        console.debug(`$ ${extScript}`);
```

Comment:
left overs?

Reply:
Tried mimicking yarn actually, where it outputs what is being ran before running it.

Reply:
ok